src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -151,13 +177,35 @@ const CreateRoomForm = ({ onFormSubmit }) => {
         )
     }
 
+    const isNumeric = (destination) => {
+        for (let char of destination) {
+            if (char >= '0' && char <= '9') continue
```

Comment:
if number start with `+` ?

Reply:
I'm not sure I got this, could you pls provide a *commit suggestion* ?

Reply:
`isNumeric` function is to check if destination is a numeric ? Right?. If destination is for example `+2336479278` it will return false. You can replace by 
```
const isNumeric = (destination) => !isNaN(Number(destination))
```

Reply:
we can apply it in another pr